assets/data/cards_pt-br.json

```
@@ -482,6 +482,72 @@
         "Conceito",
         "Back-end"
       ]
+    },
+    {
+      "title": "NaN",
+      "description": "A propriedade global NaN é um valor especial que significa Not-A-Number(Não é um número).",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Null",
+      "description": "Null é um literal que representa um valor nulo ou vazio.",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Undefined",
+      "description": "O valor global undefined representa um valor indefinido, ou seja, um valor que não foi definido ou atribuído em uma variável.",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Falsy",
+      "description": "Falsy é um valor que se traduz por falso quando avaliado em um contexto Booleano.",
+      "content": {
+        
+        "code": "São considerados valores Falsy(false, null, undefined, 0, [], '')"
+      },
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Truthy",
+      "description": "Truthy é um valor que se traduz por falso quando avaliado em um contexto Booleano.",
+      "content": {
+        
+        "code": "São considerados valores Truthy(true, Number(42, -42, 3.14, -3.14), Infinity, Strings('foo'))"
+      },
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Concatenação",
+      "description": "É o termo que se da quando juntamos um texto a outro por meio do operador (+)",
```

Veja o que acha dessa explicação, trazendo um pouco mais de contexto:

```suggestion
      "description": "Concatenação é o termo usado para descrever o processo de unir dois ou mais textos em um único texto. Em muitas linguagens de programação, você usa o operador + para fazer isso. Por exemplo, se você tem a palavra _Olá_ e a palavra _Mundo_, você pode usar + para juntar essas palavras e formar _Olá Mundo_",
```

---

Vale falar que concatenação não é só pra strings, pode ser pra listas ou outros objetos também. Em python, por exemplo, dá pra concatenar dataframes em pandas. Acho que mais importante é dizer que concatenação é um append né, no caso ele adiciona sempre no final do outro objeto.